project4/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Project 4</title>
+</head>
+<div class="nav-bar">
+    <input id="toggle" class="menu-toggle" type="checkbox" aria-hidden="true">
+    <label for="toggle" class="nav-icon" aria-hidden="true">
+        Expand the menu
+        <span class="icon-line"></span>
+        <span class="icon-line"></span>
+        <span class="icon-line"></span>
+    </label>
+    <nav>
+        <ul class="nav-items">
+            <li class="nav-item"><a href="#">home</a></li>
+            <li class="nav-item"><a href="#">about us</a></li>
+            <li class="nav-item"><a href="#">buy a ticket</a></li>
+            <li class="nav-item"><a href="#">contact us</a></li>
+            <ul class="hashtag">
```

Comment:
You need to close the first `ul` before starting a new one:
```suggestion
            </ul>  
            <ul class="hashtag">
```